core_backend/tests/api/conftest.py

```diff
@@ -99,34 +106,58 @@ async def asession(
 
 
 @pytest.fixture(scope="session", autouse=True)
-def user(client: TestClient, db_session: Session) -> None:
-    global TEST_USER_ID
-    user1_db = UserDB(
-        username=TEST_USERNAME,
-        hashed_password=get_password_salted_hash(TEST_PASSWORD),
-        hashed_api_key=get_key_hash(TEST_USER_API_KEY),
-        content_quota=TEST_CONTENT_QUOTA,
-        created_datetime_utc=datetime.utcnow(),
-        updated_datetime_utc=datetime.utcnow(),
-    )
-    user2_db = UserDB(
-        username=TEST_USERNAME_2,
-        hashed_password=get_key_hash(TEST_PASSWORD_2),
-        hashed_api_key=get_key_hash(TEST_USER_API_KEY_2),
-        content_quota=TEST_CONTENT_QUOTA_2,
+def admin_user(client: TestClient, db_session: Session) -> None:
+    admin_user = UserDB(
+        username=TEST_ADMIN_USERNAME,
+        hashed_password=get_password_salted_hash(TEST_ADMIN_PASSWORD),
+        hashed_api_key=get_key_hash(TEST_ADMIN_API_KEY),
+        content_quota=None,
+        api_daily_quota=None,
         created_datetime_utc=datetime.utcnow(),
         updated_datetime_utc=datetime.utcnow(),
     )
-    db_session.add(user1_db)
-    db_session.add(user2_db)
+
+    db_session.add(admin_user)
     db_session.commit()
+    yield admin_user.user_id
+
 
-    # update the TEST_USER_ID global variable
-    TEST_USER_ID = user1_db.user_id
+@pytest.fixture(scope="session")
+def user1(client: TestClient, db_session: Session) -> None:
+    stmt = select(UserDB).where(UserDB.username == TEST_USERNAME)
+    result = db_session.execute(stmt)
+    user = result.scalar_one()
+    yield user.user_id
+
+
+@pytest.fixture(scope="session", autouse=True)
+def user(
+    client: TestClient, db_session: Session, admin_user, fullaccess_token_admin: str
```

Comment:
Why does it use admin_user as a fixture? Do we need an Admin account to create these two?

Reply:
Yes, Because I am creating the users using the API and not using the db sesion. And since only the first user (user with user_id=1) can create users I need that users for user creation.